CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 # Change Log
 All notable changes to COMMIT will be documented in this file.
 
+
+## [1.5.0] - 2020-10-23
+
+### Added
+- core.load_confidence_map(): Load the confidence map from nii file
+
+### Changed
```

Comment:
I would add just one line under "### Added" (it's actually an added feature) and would simply write "Added the possibility to specify a voxel confidence map"

Reply:
Agree